README.md

```diff
@@ -1,2 +1,68 @@
-# sheldon
-Very Simple Erlang Spell Checker
+![](http://i.giphy.com/M3EZtLUDLsYP6.gif)
+
+# Sheldon
+
+Very Simple Erlang Spell Checker.
+
+## Contact Us
+For **questions** or **general comments** regarding the use of this library,
+please use our public [hipchat room](http://inaka.net/hipchat).
+
+If you find any **bugs** or have a **problem** while using this library, please
+[open an issue](https://github.com/inaka/sheldon/issues/new) in this repo
+(or a pull request :)).
+
+And you can check all of our open-source projects at [inaka.github.io](http://inaka.github.io).
+
+## Installation
+
+1. Clone the repo
+2. `rebar3 compile`
+
+## Usage
+
+### Erlang Shell
+
+First of all Sheldon is an application and it needs to be started. You can use `sheldon:start/0`.
```

Comment:
You should create a file to do all this, something like [`./console`](https://github.com/inaka/gadget/blob/master/console) from gadget or check how does the command [`rebar3 shell`](https://www.rebar3.org/docs/configuration) works.